draft-ietf-dnsop-dns-capture-format-01.md

```diff
@@ -719,14 +723,12 @@ of the original packet stream cannot be re-constructed from the C-DNS format:
      * A DNS payload may have be split across multiple TCP segments
      * Multiple DNS messages may have be sent on a single TCP session
 * Malformed DNS messages if they are not recorded
-* Non-DNS packets
```

Comment:
Because we don't refer to Non-DNS packets anywhere else. In the compactor these are packets that we receive but which aren't sent to port 53. The compactor calls these Non-DNS, and does not try to decode any DNS from them, so they don't get classified as malformed.

I think that this is a compactor-specific thing, and shouldn't be in the draft. Hence removing them from the statistics too.